aws-observability/json/EC2-Metrics-App.json

```diff
@@ -1894,6 +1898,15 @@
                     "keepVisualSettingsConsistentWithParent": true,
                     "panelType": "TextPanel",
                     "text": "Failure Events"
+                },
+                {
+                    "id": null,
+                    "key": "panelPANE-E40BD69ABB08F949",
+                    "title": "Note",
+                    "visualSettings": "{\"general\":{\"mode\":\"TextPanel\",\"type\":\"text\",\"displayType\":\"default\"},\"title\":{\"fontSize\":14},\"text\":{\"format\":\"markdownV2\"},\"series\":{},\"legend\":{\"enabled\":false}}",
+                    "keepVisualSettingsConsistentWithParent": true,
+                    "panelType": "TextPanel",
+                    "text": "This dashboard requires Cloud Trail Logs from AWS and are specific for AWS EC2."
```

Comment:
I would recommend
This dashboard requires AWS CloudTrail Logs from AWS account and the contents are specific for AWS EC2 instance(s).